man/systemd.socket.xml

```diff
@@ -922,6 +922,20 @@
         <xi:include href="version-info.xml" xpointer="v255"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>PassFileDescriptorsToExec=</varname></term>
+
+        <listitem><para>Takes a boolean argument. Defaults to off. If enabled, file descriptors created by
+        the socket unit are passed to <varname>ExecStartPost=</varname>, <varname>ExecStopPre=</varname>, and
+        <varname>ExecStopPost=</varname> commands from the socket unit. Mentioned <varname>ExecXYZ=</varname>
+        commands can access file descriptors with
+        <citerefentry><refentrytitle>sd_listen_fds</refentrytitle><manvolnum>3</manvolnum></citerefentry>
+        interface as if they were socket activated. Note that <varname>ExecStartPre=</varname> command cannot
```

Comment:
We don't usually say "XYZ" like this. Maybe:

> The passed file descriptors can be accessed with `sd_listen_fds`, as if the commands were invoked from the associated service units.

You get the idea ;-)

Reply:
That does sound better. Updated. Thanks.